README.md

```diff
@@ -710,6 +710,7 @@ The OpenTelemetry Operator *might* work on versions outside of the given range,
 
 | OpenTelemetry Operator | Kubernetes           | Cert-Manager        |
 |------------------------|----------------------|---------------------|
+| v0.93.0                | v1.23 to v1.29       | v1                  |
```

Comment:
it would be good to remove the last one from the table

Reply:
done